build.cmd

```diff
@@ -23,7 +23,7 @@ set "__LogsDir=%__RootBinDir%\Logs"
 set __MSBCleanBuildArgs=
 set __SkipTestBuild=
 set __ToolchainMilestone=testing
-set __DotNetCliPath=
+set "__DotNetCliPath=%__ProjectDir%\Tools\dotnetcli"
 
```

Comment:
The CLI package can be still updated independently on msbuild or buildtools, same as today. I am just changing how and where it gets downloaded to be consistent with other repos.